Mail/Helpers/RichTextEditor.swift

```diff
@@ -264,23 +264,10 @@ final class MailEditorView: SQTextEditorView {
         callEditorMethod(name: "moveCursorToStart", completion: completion)
     }
 
-    /// Removes any current selection and moves the cursor to the very end of the document.
-    func moveCursorToEnd(completion: ((_ error: Error?) -> Void)? = nil) {
-        callEditorMethod(name: "moveCursorToEnd", completion: completion)
-    }
-
-    func addBold(completion: ((_ error: Error?) -> Void)? = nil) {
-        callEditorMethod(name: "bold", completion: completion)
-    }
```

Comment:
Wondering if @valentinperignon intended on reusing those for his editor ?

Reply:
I checked with Valentin, all good for him to remove it